### ISSUE TITLE
abstract.commits.info() returns latest commit even with sha passed

### ISSUE THREAD

**Issue by danielthall** [goabstract/abstract-sdk#133]
Regardless of what SHA is passed to this request, the latest commit is always returned:

```
const c = await abstract.commits.info({
    projectId: 'a5161040-9a83-11e8-b10e-77b04fc54df0',
    branchId: 'master',
    sha: 'bf596b716b1993c175a652577c1c956873b478c1'
});
```

responds with:

```
{ sha: 'd47c72c1c4aa42fce84e7caae048f5cbd0b8836c',
  userName: 'Louie Solomon',
  userEmail: '4668cfe9-ccaf-420e-af59-017b2315d394@users.abstractapp.com',
  userId: '4668cfe9-ccaf-420e-af59-017b2315d394',
  title: 'Merged "left nav fix"',
  description: '',
  time: '2019-05-21T06:02:42.000Z',
  parents:
   [ 'bf596b716b1993c175a652577c1c956873b478c1',
     '5cf45717bf029d1b76b841b09d88b8644728a106' ],
  type: 'MERGE',
  fileIds: [],
  sourceBranchId: 'c2d16b97-ad5f-4595-9bbd-7dd8ed63fc4c',
  sourceBranchName: 'left nav fix',
  destinationBranchId: 'master',
  destinationBranchName: 'Master',
  projectId: 'a5161040-9a83-11e8-b10e-77b04fc54df0' }
```

**Comment by tommoor** [comment]
@danielthall thanks for filing, we're looking into it as a matter of priority 👍

**Comment by bitpshr** [comment]
Hi @danielthall, thanks again for taking the time to report this bug. This should be fixed as of `abstract-sdk@v1.5.1`. Please don't hesitate to let us know if you have any additional issues or questions while using the SDK.

**Comment by danielthall** [comment]
Thanks for the quick turn around! Works great!
